archunit-tooling-test/build.gradle

```diff
@@ -0,0 +1,191 @@
+import org.apache.tools.ant.filters.ReplaceTokens
+
+plugins {
+    id 'archunit.java-production-conventions'
+    id 'org.ajoberstar.grgit' version '5.0.0'
+    id "de.undercouch.download" version "5.0.2"
+}
+
+repositories {
+    flatDir {
+        dirs "${rootDir}/inlineDeps/"
+    }
+}
+
+ext.moduleName = 'com.tngtech.archunit.tooling'
+ext.minimumJavaVersion = JavaVersion.VERSION_1_9
+
+sourceSets {
+    testFixtures {
```

Comment:
I couldn't just apply `java-test-fixtures` because it gets added to the classpath in the form of a Jar, messing up the test engines, which depend on file paths inside the resources directories